packages/SketchMorph2-Core.package/MorphicMonet.class/instance/clickMenuItemWith..st

```diff
@@ -0,0 +1,7 @@
+accessing
+clickMenuItemWith: aString
+	self submenuDict valuesDo: [:submenu | 
+		submenu select: [:item | 
+			item isMenuItemMorph and: [item contents = aString]] thenDo: [:item | 
```

Comment:
Maybe it would be better to make use of the `Morph` classes name property like the mtf [does](https://github.com/hpi-swa-teaching/Morphic-Testing-Framework#mtfmorphwrapperfindbyname-astring) (which we already use, e.g. in the `M2BrushSizeSlider` class).

Reply:
We added a test checking that all `contents` are unique so that this will always work. We'd have the same issue of possible duplicates with the `name` property as we now could have with `contents`.